lib/mirage.ml

```diff
@@ -1912,6 +1912,8 @@ module Project = struct
     "open Lwt.Infix\n\
      let return = Lwt.return\n\
      let run = OS.Main.run"
+  let packages = [package "mirage"]
```

Comment:
should this then be a `build:true` dependency?  and I'm not sure what the benefit of adding it here is (apart from being exact, which I appreciate :)) -- if you run `mirage configure` or `mirage build`, you will have to have `mirage` installed!?